C/plugins/north/OMF/linkdata.cpp

```diff
@@ -366,7 +392,7 @@ bool OMFLinkedData::flushContainers(HttpSender& sender, const string& path, vect
 	catch (const BadRequest& e)
 	{
 
-		Logger::getLogger()->warn("Sending containers, not blocking issue: %s - %s %s",
+		Logger::getLogger()->warn("The OMF endpoint reported a bad request when sending containers: %s - %s %s",
```

Comment:
In branch FOGL-7427, I replaced the _path_ argument with _m_sender.getHTTPResponse()_ in all of these error logs. I thought dumping the HTTP response was more useful than repeating the path.

Reply:
I agree the path is not very useful. Will the user understand the HTTP response?

Reply:
No, but we might. This is a stop-gap measure until I can finish properly parsing of the HTTP response.